lib/Parse/ParseDecl.cpp

```diff
@@ -6594,21 +6610,39 @@ Parser::parseDeclPrecedenceGroup(ParseDeclOptions flags,
       // We want to continue parsing after this.
       invalid = true;
     }
-    attrKeywordLoc = consumeToken(tok::identifier);
+    attrKeywordLoc = consumeToken();
```

Comment:
Why did this change?

Reply:
I guess I changed it to the shortcut version because we only call this lambda after we make sure it's an identifier, but yes, it isn't related to this PR and it's safer with the check, so I'll revert that.